pkg/inject/sidecar.go

```diff
@@ -485,3 +513,21 @@ func isContainerPortAvailable(port int32, dep *appsv1.Deployment) bool {
 	}
 	return true
 }
+
+// GetConfigMapsMatchedEnvFromInDeployment return configMap which matches with configMapRef
+func GetConfigMapsMatchedEnvFromInDeployment(dep appsv1.Deployment, configMaps []corev1.ConfigMap) []corev1.ConfigMap {
+	configMapSearchMap := make(map[string]corev1.ConfigMap)
+	for _, cm := range configMaps {
+		configMapSearchMap[cm.Name] = cm
+	}
+
+	matchedConfigMaps := []corev1.ConfigMap{}
+	for _, container := range dep.Spec.Template.Spec.Containers {
+		for _, envConfigMap := range container.EnvFrom {
+			if matchedcm, ok := configMapSearchMap[envConfigMap.ConfigMapRef.Name]; ok {
```

Comment:
```suggestion
			if matchedCM, ok := configMapSearchMap[envConfigMap.ConfigMapRef.Name]; ok {
```
Always use camelCase for variable names.